### PR TITLE
Implement `libwebp` compilation support

### PR DESCRIPTION
**Pull request by mdrokz**
# Changes

- Implements [libwebp](https://developers.google.com/speed/webp) for ffmpeg
- Implemented --enable-webp or -webp flag to enable compilation

### CODE REVIEW

File: scripts/libwebp/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+export CC=${FAM_CC}
```
Comment (Javernaut) [inline scripts/libwebp/build.sh:3]: Please pass the compiler to the configure script directly. The build.sh files are executed with the `source` command, which may modify the outer environment and the CC variable may affect all subsequent build scripts (even for other ABIs).

**Review by Javernaut — CHANGES_REQUESTED** [review]

**Comment by Javernaut** [comment]
Hello @mdrokz ,

Please also retarget the PR to the `development` branch. I recently added a trigger for the Github Actions, so the PR should be picked up by the Compilability Check.

File: scripts/libwebp/build.sh (hunk 1 above)
Comment (mdrokz) [reply scripts/libwebp/build.sh:3]: > Please pass the compiler to the configure script directly. The build.sh files are executed with the `source` command, which may modify the outer environment and the CC variable may affect all subsequent build scripts (even for other ABIs).

I see, i will change that

**Comment by mdrokz** [comment]
> Hello @mdrokz ,
> 
> Please also retarget the PR to the `development` branch. I recently added a trigger for the Github Actions, so the PR should be picked up by the Compilability Check.

Sure.

File: scripts/libwebp/build.sh (hunk 1 above)
Comment (mdrokz) [reply scripts/libwebp/build.sh]: Done 388294b

Comment (Javernaut) [reply scripts/libwebp/build.sh]: I actually ment passing the CC directly to the configure script like [this](https://github.com/Javernaut/ffmpeg-android-maker/blob/master/scripts/libfreetype/build.sh#L21).

Comment (mdrokz) [reply scripts/libwebp/build.sh]: > I actually meant passing the CC directly to the configure script like [this](https://github.com/Javernaut/ffmpeg-android-maker/blob/master/scripts/libfreetype/build.sh#L21).

my bad, i changed it now

**Review by Javernaut — APPROVED** [review]

**Comment by Javernaut** [comment]
Thanks for the contribution. I'll modify the script slightly to skip some redundant parts later.

**Pull request merged**